File: In_Ruby/ex8.rb
def yakuniy_im(ball, imt)
  if imt === "Yakuniy imtihon"
    ball >= 89 && ball <= 100
  else
    ball >= 90
  end
  "Bu imtihon javobi chiqmagan"
end


puts yakuniy_im(95, "Nazorat imtihon")

File: In_Javascript/ex8.js
// Oson yo'l bilan yechganda
function yakuniy_im(ball, imt) {
    if (imt === "Yakuniy imtihon") {
        return ball >= 89 && ball <= 100;
    } else {
        return ball >= 90;
    }
}

console.log(yakuniy_im(89, "Yakuniy imtihon"));